README.md

```diff
@@ -202,7 +203,16 @@ cordova.plugins.email.requestPermission(permission, callbackFn);
 ```
 
 __Note:__ The author of the app has to make sure that the permission is listed in the manifest.
-
+For __Android__ you may add the following lines to config.xml to achieve this: 
```

Comment:
this is breaking the build when I insert this in `config.xml`, did you mean `edit-config` instead?

Reply:
```
> Task :app:mergeDebugResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeDebugResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Android resource compilation failed
     /home/joao/dev/form-for-parking-violation/platforms/android/app/src/main/res/xml/config.xml:97: AAPT: error: unbound prefix.
         
     /home/joao/dev/form-for-parking-violation/platforms/android/app/src/main/res/xml/config.xml: AAPT: error: file failed to compile.
```

Reply:
I could make it work, it was missing `xmlns:android="http://schemas.android.com/apk/res/android"` at the `widget` root of `config.xml`